src/System.Net.Requests/tests/HttpWebRequestTest.cs

```diff
@@ -1161,21 +1076,6 @@ public void Method_SetInvalidString_ThrowsArgumentException(Uri remoteServer)
             Assert.Throws<ArgumentException>("value", () => request.Method = "Method(2");
         }
 
-        [Fact]
```

Comment:
Those were the tests I created with a previous PR. It made sense to combine them into one method so that instead of ~10 requests we only make one and test it in there. See L181